packages/flutter_tools/lib/src/commands/generate_localizations.dart

```diff
@@ -200,6 +200,10 @@ class GenerateLocalizationsCommand extends FlutterCommand {
       help:
           'When specified, the "dart format" command is run after generating the localization files.',
     );
+    argParser.addMultiOption(
```

Comment:
Let's not add this.

All of the options that are important to `dart format` can now be specified by the user:

https://dart.dev/tools/dart-format#configuring-formatter-page-width